rfc-035-use-standard-linting-tools-across-all-our-projects.md

```diff
@@ -0,0 +1,39 @@
+# Use standard linting tools across all of our projects
+
+## Summary
+
+Where available, all new dxw-owned or maintained repositories SHOULD use a
```

Comment:
At the moment this just says that _a_ linter should be used, rather than the specific linter we've picked for that language, so I think we can make this language a bit stronger.

Reply:
We should probably also add an option to break this rule with an ADR for situations where there is a good reason not to follow it (eg. the client has a linter they prefer)

Reply:
I think use of "SHOULD" allows for breaking the rule if needed.

Reply:
Also, I think this RFC is "use any linter". I'd welcome RFCs for specific linter choices, but I don't think it needs to be part of this RFC.